### DISCUSSION TITLE
How to change "Acronyms" section name

### DISCUSSION THREAD

**Discussion by mpd-azevedo** [joaomlourenco/novathesis#461] [Q&A] [answered]
Hello, 

I'm using the NovaThesis template and I would like to change the "Acronyms" section to a different name but I can't seem to figure out how. I tried changing the name in Config > files.tex and in Chapters > Acronyms but without success.

I would appreciate some help

Thank you very much

**Comment by joaomlourenco** [comment, marked as answer]
In the file `Config/5_packaes.tex` add
```latex
\addto{\captionsenglish}{
  \renewcommand{\acronymname}{YOUR NAME HERE}
}
```

**Reply by mpd-azevedo** [reply]
It worked well,

Thank you very much for the quick answer, Professor
